README.md

```diff
@@ -61,10 +61,123 @@ By default the following dependencies are enabled:
 - [Strimzi Kafka Operator](https://strimzi.io/)
 - [Cert Manager](https://github.com/cert-manager/cert-manager)
 
+
+
+### Deploy in EKS Cluster
+
+To deploy the Big Peer into a AWS EKS Cluster like EKS you will need to perform the following actions;
+
+#### EKS Cluster Setup Checks
+
+- Create the [EKSClusterRole](https://docs.aws.amazon.com/eks/latest/userguide/cluster-iam-role.html#create-service-role)
+
+
+- Install [eksctl](https://eksctl.io/installation/)
+
+- Deploy EKS cluster with eksctl command
+
+<br/>
+
+```shell
+eksctl create cluster --name <cluster-name> --region <aws-region> --profile <aws-profile>
+```
+<br/>
+
+- Once the cluster is up and running verify that you have [ebs-csi-driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html#managing-ebs-csi) running in the `kube-system` namespace. This ensures that you have the provisioner that will dynamicallly create the Persistent Volumes that the hydra-store and kafka zookeeper nodes will need.
+
+<br/>
+
+- If your default storageClass is not set as the the default storageClass you can do that by adding the line below to the annotations.
+
+```yaml
+  storageclass.kubernetes.io/is-default-class: "true"
+```
+
+<br/>
+
+- Install and Configure and [ingress controller](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/README.md)(***We recommend ingress-nginx***) or [AWS LoadBalancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller). 
+
+- Make sure you have access to the required registry to pull images from the helm chart. This can be done by create a secret that has the registry username,email and password/token.
+
+<br/>
+
+#### Helm Chart Ingress Config 
+
+- In order to be able to access the big peer externally you need to configure the ingress in the values.yaml with a valid host name. To do this you will need to create a **CNAME** record that points to the loadbalancer created in the ingress service that has the type `LoadBalancer`.
+
+- Then in your values.yaml file you will need to enable the ingress and update the `host` value with the hostname you created as shown below;
+
+```yaml
+ingress:
+  main:
+    # -- Enables or disables the ingress
+    enabled: true
+
+    # -- Make this the primary ingress (used in probes, notes, etc...).
+    # If there is more than 1 ingress, make sure that only 1 ingress is marked as primary.
+    primary: true
+
+    # -- Override the name suffix that is used for this ingress.
+    nameOverride:
+
+    # -- Provide additional annotations which may be required.
+    annotations:
+      {}
+      # kubernetes.io/ingress.class: traefik
+      # kubernetes.io/tls-acme: "true"
+
+    # -- Provide additional labels which may be required.
+    labels: {}
+
+    # -- Set the ingressClass that is used for this ingress.
+    # Requires Kubernetes >=1.19
+    ingressClassName: nginx # "traefik"
```

Comment:
same as above lets use the native eks stack for ingress https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html

Reply:
Yeah would want me use the AWS Load Balancer Controller instead?